### DISCUSSION TITLE
Quanto mais visita melhor?

### DISCUSSION THREAD

**Discussion by Rich14S** [Rich14S/OMG_BADGES#5] [Q&A] [answered]
Ou o que importam são as estrelas?

**Comment by roodriiigooo** [comment, marked as answer]
O que importa é estudar e se desenvolver profissionalmente! :D

**Reply by arthurr-jpg** [reply]
O importante é crescer.

**Comment by arthurr-jpg** [comment]
Continuously develop and grow
